### PR TITLE
fuzzer crash fix: Null-dereference WRITE

### PR DESCRIPTION
**Pull request by qxliu76**
fix for fuzzer crash:   https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=18363

### CODE REVIEW

File: src/hb-ot-layout-common.hh [hunk 1]
```diff
@@ -1545,6 +1545,7 @@ struct ClassDefFormat2
     range_rec.value = prev_klass;
 
     RangeRecord *record = c->copy (range_rec);
+    if (unlikely (!record)) return_trace (false);
```
Comment (ebraminio) [inline src/hb-ot-layout-common.hh]: Aren't the other null checks unlikely also?

**Comment by ebraminio** [comment]
Please add the test case also, thanks

**Comment by qxliu76** [comment]
> Please add the test case file also, thanks

added. :)

**Comment by ebraminio** [comment]
Thanks :)

**Pull request merged**